Swift/ReplicatorConfiguration.swift

```diff
@@ -322,7 +322,7 @@ public struct ReplicatorConfiguration {
         
         if let resolver = self.conflictResolver {
             c.setConflictResolverUsing { (conflict) -> CBLDocument? in
-                return resolver.resolve(conflict: Conflict(impl: conflict))?._impl
+                return resolver.resolve(conflict: Conflict(impl: conflict, db: self.database))?._impl
```

Comment:
The idea is to get the collection from the replicator's config and pass to the Conflict here.